Polyfrost/Hytils-Reborn
# Fix Remove Armorstand Nametags does not fully work when Patcher's entity culling is enabled

## Description

- Fix Remove Armorstand Nametags does not fully work when Patcher's entity culling is enabled
- Update fallback LanguageData

## Related Issue(s)

Fixes #13 

## How to test

1. Turn on the Hytils and Patcher features
2. Go behind a wall or some place where the armor stand is no longer visible and make sure the tag is no longer visible

## Release Notes

```release-note
- Fix Remove Armorstand Nametags does not fully work when Patcher's entity culling is enabled
```
